wand/image.py

```diff
@@ -2182,6 +2182,10 @@ def composite(self, image, left, top):
         :type left: :class:`numbers.Integral`
         :param top: the y-coordinate where `image` will be placed
         :type top: :class:`numbers.Integral`
+        :param operator: the operator that affects how the composite
+                         is applied to the image.  available values
+                         can be found in the :const:`COMPOSITE_OPERATORS`
+                         list
```

Comment:
This new parameter has to be added into docs/changes.rst and the bottom of this docstring using `.. versionadded:: 0.4.4`.  Please see other uses of `.. versionadded::` directive.